test/integration_test_helper.rb

```diff
@@ -98,7 +98,7 @@ def select2(value, attrs)
 
   def wait_for_ajax
     Timeout.timeout(Capybara.default_max_wait_time) do
-      loop until page.evaluate_script('jQuery.active').zero?
+      loop until page.evaluate_script('jQuery.active').zero? && page.evaluate_script('window.axiosActive').zero?
```

Comment:
can you please extract this to a pr by it self? I think it would be easier to to merge quickly.

Reply:
sure